src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/sampler/HTTPHCAbstractImpl.java

```diff
@@ -186,4 +194,20 @@ protected static boolean isStaticProxy(String host){
     protected static boolean isNullOrEmptyTrimmed(String value) {
         return JOrphanUtils.isBlank(value);
     }
+
+    /**
+      * Invokes {@link HTTPAbstractImpl#getResponseTimeout()}
```

Comment:
This comment does not make things easier to understand what the method does.
It would probably move the description from `@return` section to the description.

Reply:
Done , thanks!

Reply:
Would you please remove `Invokes {@link HTTPAbstractImpl#getResponseTimeout...`?

Reply:
done!